### DISCUSSION TITLE
Wordmat virker ikke

### DISCUSSION THREAD

**Discussion by Miklosrbm** [Eduap-com/WordMat#245] [Support]
Når jeg downloader Wordmat (med eller uden antivirus) og herefter forsøger at foretage en beregning, får jeg først "Denne funktion kræver Wordmat+" og derefter "Der skete en fejl. Wordmat er formentlig ikke installeret korrekt".

Jeg har tidligere haft wordmat hvor det har virket glimrende for mig.

**Comment by IBG-IT** [comment]
fjern og installer igen - så virkede det for mig .-)
